intellij-plugin/src/main/resources/inspectionDescriptions/ApolloMissingGraphQLDefinitionImport.html

```diff
@@ -0,0 +1,19 @@
+<html>
+<body>
+Reports usages of directives that are not imported.
```

Comment:
```suggestion
Reports usages of types and directives that are not imported.
```

Reply:
Done in 2245274:


https://github.com/apollographql/apollo-kotlin/assets/372852/40898316-bd62-4f0d-8105-e184f22f62ad